src/components/transactions.js

```diff
@@ -46,29 +33,54 @@ export default class Transactions extends React.Component {
     }
 
     calcOutflow(transactions) {
-        var inflow = 0;
+        var outflow = 0;
         for (var transaction of transactions) {
             if (transaction.spent) {
-                inflow += transaction.amount
+                outflow += transaction.amount
             }
         }
-        return inflow
+        return outflow
+    }
+
+    addNewTransaction(category, amount, spent) {
+        
+        const newTransaction = {id: this.state.transactions.length + 1, spent: spent, date: new Date().getDate() + '.' + new Date().getMonth() + '.' + (new Date().getFullYear() - 2000), category: category, amount: amount}
```

Comment:
why not just sending `new Date()` and later on just formatting as you need when you display it?